cuda_bindings/tests_cython/test_ccuda.pyx

```diff
@@ -5,6 +5,9 @@
 # this software. Any use, reproduction, disclosure, or distribution of
 # this software and related documentation outside the terms of the EULA
 # is strictly prohibited.
+
+# distutils: language=c++
```

Comment:
I struggled because I did not bother check the diffs of the test code, until now. It turns out that this was added in #252, and was why we use `CPLUS_INCLUDE_PATH` instead of `C_INCLUDE_PATH`. A naive backport missed this context.